website/content/api-docs/system/remount.mdx

```diff
@@ -12,7 +12,10 @@ The `/sys/remount` endpoint is used remount a mounted backend to a new endpoint.
 
 ## Move Backend
 
-This endpoint moves an already-mounted backend to a new mount point.
+This endpoint moves an already-mounted backend to a new mount point. 
+
+~> Note: This endpoint requires a policy with both `sudo` and `update` capabilities to `sys/remount`
```

Comment:
Are you sure about sudo?  I'm not seeing it in the code.

Reply:
```
mike-C02ZC5TSLVDL:~ mike$ vault policy read remount-policy
path "sys/remount" {
  capabilities = ["update"]
}
mike-C02ZC5TSLVDL:~ mike$ vault token create -policy=remount-policy
Key                  Value
---                  -----
token                s.RnHjENBvc8beHpvlniQjkdi5
token_accessor       Qq0XetUk5PUlPl9VMHNczQYU
token_duration       72h
token_renewable      true
token_policies       ["default" "remount-policy"]
identity_policies    []
policies             ["default" "remount-policy"]
mike-C02ZC5TSLVDL:~ mike$ VAULT_TOKEN=s.RnHjENBvc8beHpvlniQjkdi5 vault secrets move kvE/ kvE-moved/
Error moving secrets engine kvE/ to kvE-moved/: Error making API request.

URL: POST https://vault.greenacreslimited.com:8200/v1/sys/remount
Code: 403. Errors:

* 1 error occurred:
	* permission denied


mike-C02ZC5TSLVDL:~ mike$ vault policy read remount-policy
path "sys/remount" {
  capabilities = ["update", "sudo"]
}

mike-C02ZC5TSLVDL:~ mike$ vault token create -policy=remount-policy
Key                  Value
---                  -----
token                s.c4ckIvYP6NV6OzgNJDcAkA9m
token_accessor       YShBcGn8eYq6MG8xs7hQyTcu
token_duration       72h
token_renewable      true
token_policies       ["default" "remount-policy"]
identity_policies    []
policies             ["default" "remount-policy"]

mike-C02ZC5TSLVDL:~ mike$ VAULT_TOKEN=s.c4ckIvYP6NV6OzgNJDcAkA9m vault secrets move kvE/ kvE-moved/
Success! Moved secrets engine kvE/ to: kvE-moved/
```

Reply:
Ah, I forgot how to identify sudo paths, my mistake.